depmap_omics_upload/tracker.py

```diff
@@ -122,14 +115,12 @@ def write_seq_table(self, df):
         self.mapping_utils.rename_columns(
             self.seq_table_name, df, convert_to_custom_names=False, inplace=True
         )
-        self.client.update(self.seq_table_name, df)
-        self.client.commit()
+        self.client.update_only(self.seq_table_name, df)
 
     def write_str_table(self, df):
         df.index.name = self.str_table_index
         df = df.reset_index(level=0)
-        self.client.update(self.str_table_name, df)
-        self.client.commit()
+        self.client.update_only(self.str_table_name, df)
```

Comment:
Does `update_only` prevent insertion/deletion of rows? Is this intentional?

Reply:
Yes, it can only update existing rows. Did `update` function as an upsert in the past? Only `insert_only` and `update_only` are implemented in the current client.

Reply:
Yeah I think it used to allow upsert. There was also an `delete_missing_rows` flag in the old client but looks like we never turned it on anyways... I think it's fine to leave it as `update_only` though. We have separate insert functions anyways